internal/olm/fbcutil/util.go

```diff
@@ -48,7 +48,7 @@ const (
 	// TODO(v2.0.0): pin this image tag to a specific version.
 	DefaultIndexImage = DefaultIndexImageBase + "latest"
 	// DefaultInitImage is the default image to be used in the registry init container
-	DefaultInitImage = "docker.io/library/busybox:1.36.0"
+	DefaultInitImage = "registry.access.redhat.com/ubi8-micro:8.9"
```

Comment:
`ubi-micro`, nor `ubi-mimial` have contain `gzip` see the below:

https://catalog.redhat.com/software/containers/ubi8/ubi-micro/5ff3f50a831939b08d1b832a?architecture=amd64&image=66042656ca075e30d8ca87ad&container-tabs=packages

https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8?architecture=amd64&image=660383f31ba64b6bd44df0a7&container-tabs=packages

If this was to change it would have to be the full `ubi` image.

```
podman run -it --rm --entrypoint /bin/bash registry.access.redhat.com/ubi8:8.9
Trying to pull registry.access.redhat.com/ubi8:8.9...
Getting image source signatures
Checking if image destination supports signatures
Copying blob 2efec45cd878 done   | 
Copying config c70d72aaeb done   | 
Writing manifest to image destination
Storing signatures
[root@ffdbee72b4c1 /]# cat --version
cat (GNU coreutils) 8.30
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Torbjorn Granlund and Richard M. Stallman.
[root@ffdbee72b4c1 /]# gzip --version
gzip 1.9
Copyright (C) 2017 Free Software Foundation, Inc.
Copyright (C) 1993 Jean-loup Gailly.
This is free software.  You may redistribute copies of it under the terms of
the GNU General Public License <https://www.gnu.org/licenses/gpl.html>.
There is NO WARRANTY, to the extent permitted by law.

Written by Jean-loup Gailly.
```

Reply:
fixed, thank you @acornett21